src/ovp_pipeline/ui/view_models.py

```
@@ -4265,6 +4307,78 @@ def build_graph_map_payload(
+            # the active theme.  Cycle through 1..8 by index.
+            "color": f"var(--c-{(idx % 8) + 1})",
+            "count": int(cluster.get("member_count") or 0),
+        }
+        for idx, cluster in enumerate(clusters)
+    ]
+    # ``absorbedAt`` is required by the kit's timeline scrubber.
+    # We don't yet surface a per-object absorbed-at on the cluster
+    # member dict; for v1 every node carries today's date so the
+    # timeline degenerates to a single bucket and the "Play history"
+    # affordance is harmless.  Stage 4 will surface real timestamps.
+    absorbed_default = _dt.date.today().isoformat()
```

Ruff (0.15.12)

[warning] `datetime.date.today()` used

(DTZ011)